cinema_service/settings.py

```diff
@@ -86,8 +91,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("POSTGRES_DB"),
```

Comment:
It's a good practice to use `os.getenv` instead of `os.environ.get` because it allows to set a default value if the environment variable is not set. For example: `os.getenv('POSTGRES_DB', 'default')`.